ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/aaa-server-groups.j2

```diff
@@ -8,7 +8,8 @@
 | ------------------| ----- | --- | ---------- |
 {%     for aaa_server_group in aaa_server_groups %}
 {%         for server in aaa_server_group.servers %}
-| {{ aaa_server_group['name'] }} | {{ aaa_server_group['type'] }} | {% if server['vrf'] is defined and server['vrf'] is not none %} {{ server['vrf'] }} {% else %} default {% endif %}| {{ server['server'] }} |
+{%             set vrf = server['vrf'] | arista.avd.default('default') %}
```

Comment:
We should use jinja style when the key is a static string value, so `set vrf = server.vrf ...` same below

Reply:
same for other templates

Reply:
Updated.